ui/src/flux/components/DatabaseListItem.tsx

```diff
@@ -83,6 +81,44 @@ class DatabaseListItem extends PureComponent<Props, State> {
     })
   }
 
+  private get filterAndTagList(): JSX.Element {
+    const {db, service} = this.props
+    const {isOpen, searchTerm} = this.state
+
+    if (isOpen) {
+      return (
+        <>
+          <div className="flux-schema--filter">
+            <input
+              className="form-control input-sm"
+              placeholder={`Filter within ${db}`}
+              type="text"
+              spellCheck={false}
+              autoComplete="off"
+              value={searchTerm}
+              onClick={this.handleInputClick}
+              onChange={this.onSearch}
+            />
+          </div>
+          <TagList db={db} service={service} tags={this.tags} filter={[]} />
+        </>
+      )
+    }
+  }
+
+  private handleCopyClick = e => {
```

Comment:
minor, but what do you think about calling this `handleClickCopy` to indicate that it's handling an action, and then the content of that action? that's generally the idiomatics of such methods elsewhere in the codebase, to my recollection

Reply:
👍